.github/workflows/buildbot.yml

```diff
@@ -7,17 +7,23 @@ on:
     branches: [ master ]
 
 jobs:
-  build:
-
+  check:
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up JDK 11
-      uses: actions/setup-java@v1
-      with:
-        java-version: 11
-    - name: Grant execute permission for gradlew
-      run: chmod +x gradlew
-    - name: Check with Gradle
-      run: ./gradlew check
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle.kts') }}
+          restore-keys: ${{ runner.os }}-gradle
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Check with Gradle
+        run: ./gradlew --no-daemon check jacocoTestReport coverallsJacoco
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This should work fine, but may need to be changed to `secrets. COVERALLS_REPO_TOKEN` (see [instructions](https://github.com/nbaztec/coveralls-jacoco-gradle-plugin#github-actions)) if the build fails. Theoretically the `GITHUB_TOKEN` version should allow it to work for forks, too — and it does on my fork.

Reply:
Yup, it works, yay!